views/update/update-beneficial-owner-bo-mo-review.html

```diff
@@ -20,9 +20,9 @@ <h1 class="govuk-heading-xl">You're about to review beneficial owner and managin
       <p class="govuk-body">You'll need to:</p>
       <ul class="govuk-list govuk-list--bullet">
         <li>check the information is correct, and update it if needed</li>
-        <li>enter the date of birth and home address for individual beneficial owners and managing officers</li>
-        <li>tell us if they are no longer a beneficial owner or managing officer</li>
-        <li>add any new beneficial owners or managing officers</li>
+        <li>enter the home address for individual beneficial owners and managing officers</li>
```

Comment:
no "enter the date of birth for individual beneficial owners" 
Also, what about other types of BOs?

Reply:
Just matching it up to the screenshot on the ticket. Here it is for reference :)

<img width="514" alt="image" src="https://user-images.githubusercontent.com/120723588/231717485-96d0a459-60fe-4ea3-a02b-df350546e1ab.png">